### PR TITLE
discord: 0.0.18 -> 0.0.19

### PR DESCRIPTION
**Pull request by x10an14**
As of this commit date, v0.0.18 won't let you start up - the auto-check
for available updates aborts application startup.

###### Description of changes
Simpe version update.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rissson** [comment]
I can confirm this works on `x86_64-linux`

**Review by rissson — APPROVED** [review]

**Review by skykanin — APPROVED** [review]
LGTM. Can confirm this works on `x86_64-linux`

**Comment by kristoff3r** [comment]
How many approvals do the discord version bumps need? It's broken when it happens, and it's annoying being blocked even when tracking master.

**Review by zarelit — APPROVED** [review]

**Comment by x10an14** [comment]
> How many approvals do the discord version bumps need? It's broken when it happens, and it's annoying being blocked even when tracking master.

I think we're just waiting for someone who can merge at this point? I sure can't.

**Comment by kristoff3r** [comment]
> > How many approvals do the discord version bumps need? It's broken when it happens, and it's annoying being blocked even when tracking master.
> 
> I think we're just waiting for someone who can merge at this point? I sure can't.

Ah, I thought someone with merge rights would have seen it at this point.

**Comment by OPNA2608** [comment]
> It's broken when it happens, and it's annoying being blocked even when tracking master.

As a workaround, you can tell Discord to ignore the available update and keep using it until the bump by adding `"SKIP_HOST_UPDATE": true` to your `~/.config/discord/settings.json`.

**Comment by x10an14** [comment]
> > It's broken when it happens, and it's annoying being blocked even when tracking master.
> 
> As a workaround, you can tell Discord to ignore the available update and keep using the outdated version until the bump by adding `"SKIP_HOST_UPDATE": true` to your `~/.config/discord/settings.json`.

How'd you learn of this one? =D

**Review by ldesgoui — APPROVED** [review]

**Comment by OPNA2608** [comment]
> How'd you learn of this one? =D

I don't remember how I stumbled upon the issue but I got it from https://github.com/snapcrafters/discord/issues/105#issuecomment-849165614, it's been pinned in one of my servers' linux channel ever since :wink:.

**Comment by winterqt** [comment]
Just fixed the commit message (we don't use the `v` prefix in commit messages).

Result of `nixpkgs-review pr 185953` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>discord</li>
  </ul>
</details>

**Review by winterqt — APPROVED** [review]

**Comment by winterqt** [comment]
For the future: you can always use an [overlay](https://nixos.org/manual/nixpkgs/stable/#chap-overlays) to update the package while it's not in (un)stable.

**Comment by x10an14** [comment]
> Just fixed the commit message (we don't use the `v` prefix in commit messages).
> 
> Result of `nixpkgs-review pr 185953` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
> 1 package built:
> 
>     * discord

Thanks for showing how it should be done next time =)

**Pull request merged**
